### PR TITLE
Display 1-hr-fire size for HFI

### PR DESCRIPTION
**Pull request by conbrad**
https://wps-pr-1280.apps.silver.devops.gov.bc.ca/hfi-calculator

- Hook up 1 HR fire size to HFI
- Notify user of grass type/cure missing error

- [x]  Given I am logged into the HFI calc, When I look at the daily view table, Then the 1 HR fire size column is populated for every station displayed
- [x] Given a station with a grass fuel type and no grass cure set, When I look at the daily view table, I see a red error icon and tooltip text on the grass cure cell.
- [x]  Given a station with a grass fuel type and no grass cure set, When I look at the daily view table, I see a red icon with hover text on the Mean Fire Intensity Group column (zone row).
- [x] Tests

Tess' Feedback:
1. [x] Copy fix for tooltip
2. [x] Styling fixes 
3. [x] Empty cells for ROS, Fire Size, Fire Type, HFI, FIG when grass cure missing
4. [x] Icon cell alignment 
5. [x] No error icons for missing data in the future

### CODE REVIEW

**Comment by tessmgood** [comment]
1. There's a mistake in my copy for the Mean FIG hover: Grass Cure % not defined in WFWX **for** one or more stations.
2. I know you pointed out the CSS gremlins in the planning area row - also mentioning that A made some changes to the planning area row background colour that you may need to be aware of
3. When there is a grass cure error, I expected ROS, Fire Size, Fire Type, HFI, FIG cells to be empty, but ROS and HFI are 0, fire size is a massive number, Fire Type and FIG have values. We're saying we can't calculate these in the error message, but that isn't true in the interface. 
3. Can we centre-middle align the error icons? Looks to be top left 
4. If we go into the future when there isn't any forecasted data yet (empty table), the Mean FIG error is displaying in all planning area rows. It's saying there's a grass cure missing, but that's not quuuiite a helpful or accurate error in this situation. Can we look at the logic and only display the Mean FIG error when there is a grass cure error displaying in a station row? 
Screen Shot 2021-09-02 at 3 58 47 PM
- I'm wondering if we should do some flow charting to identify different errors that are predictable, and ensure the correct errors are displaying at the right time. I can get this started but will need some dev support

**Review by Sybrand — APPROVED** [review]
Looks good - BUT - and I don't know when this sneaked in, copy paste to spreadsheet is now properly broken - the headings mess up.

File: web/cypress/integration/hfi-calculator-page.spec.ts [hunk 1]
```diff
@@ -1,49 +1,87 @@
...
+    })
+  })
+  describe('high intensity', () => {
+    beforeEach(() => {
+      cy.intercept('GET', 'api/hfi-calc/daily*', { fixture: 'hfi-calc/dailies-high-intensity.json' }).as('getDaily')
+      cy.intercept('GET', 'api/hfi-calc/fire-centres', {
+        fixture: 'hfi-calc/fire-centres-minimal.json'
+      }).as('getFireCentres')
+    })
+    it('should highest intensity values for mean intensity group in Daily View Table', () => {
```
Comment (andrea-williams) [inline web/cypress/integration/hfi-calculator-page.spec.ts:76]: the grammar in this sentence does not compute

File: web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    }
+  }
+})
+
+const toolTipFirstLine = 'Grass Cure % not defined in WFWX for one or more stations.'
+const toolTipSecondLine = 'Cannot calculate Mean FIG.'
+const toolTipElement = (
+  <div>
+    {toolTipFirstLine}
+    {toolTipSecondLine}
```
Comment (andrea-williams) [inline web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx:67]: Won't this need a `<br />`?

**Review by andrea-williams — APPROVED** [review]
Couple of random comments, but overall LGTM

File: web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx (hunk 2 above)
Comment (conbrad) [reply web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx:67]: It looks worse without, it was a `<br />` a couple of commits up.

File: web/cypress/integration/hfi-calculator-page.spec.ts (hunk 1 above)
Comment (conbrad) [reply web/cypress/integration/hfi-calculator-page.spec.ts]: Fixed

File: web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx (hunk 2 above)
Comment (conbrad) [reply web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx:67]: https://github.com/bcgov/wps/pull/1280/commits/3e61ddbb8be3ba80a2e1845b0940ac3c8bbbba61

**Pull request merged**